integration-tests/reactive-pg-client/src/test/resources/application-tl.properties

```diff
@@ -0,0 +1,6 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=hibernate_orm_test
+quarkus.datasource.password=hibernate_orm_test
+quarkus.datasource.reactive.url=${reactive-postgres.url}
+quarkus.datasource.reactive.thread-local=true
```

Comment:
The configuration property isn't suggesting that it relates to an enhanced connection pool.

Maybe `threadlocal-pool` ? Or `pool-strategy`= [enum: "threadlocal" | "simple" ] ?

Reply:
Fair enough, but `max-size` also relates to the pool without saying it, so…

Reply:
@gsmet how would you name it? You recently refactored all datasource configuration right?

Reply:
I've un-blocked the PR. Let's go ahead with the feature as we need it asap for benchmarking, but I'd still like to revisit the configuration in a follow up.